### ISSUE TITLE
“Changes updated” message is not apparent to VoiceOver users [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#269]
Upon confirming changes on the App Settings -> My Health Log screen, the status message “Changes updated” is displayed at the top of the main content of the screen. It is not immediately apparent to VoiceOver users, as it is not announced. Focus is moved to the top of the screen after form submission. The “COVID ALERT NY” annoucement may lead VoiceOver users to believe they have been taken to a new screen.

![changes-updated-message](https://user-images.githubusercontent.com/4437084/92059490-590e3200-ed5f-11ea-86aa-950f8c7e572e.png)

I suggest moving focus to the message upon its appearance.

**Comment by floridemai** [comment]
@willslone This should be fixed in the next buiild.
